.github/workflows/release.yml

```
@@ -43,6 +51,10 @@ jobs:
       - name: Set Aztec version
         run: |
           VERSION=${AZTEC_VERSION} aztec-up
+
+      - name: Update package version
+        run: |
+          npm version ${{ github.event.inputs.version }} --no-git-tag-version --allow-same-version
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)